streaming/index.js

```diff
@@ -662,9 +690,6 @@ const startServer = async () => {
    * @returns {SubscriptionListener}
    */
   const streamFrom = (ids, req, output, attachCloseHandler, destinationType, needsFiltering = false) => {
-    const accountId = req.accountId || req.remoteAddress;
-
-    log.verbose(req.requestId, `Starting stream from ${ids.join(', ')} for ${accountId}`);
```

Comment:
This log line ended up being duplicated due to changes to the initialisation logging.